src/library/injections/dmm_takeover.js

```diff
@@ -567,13 +572,13 @@
 					$(".overlay_idle").css({ background: 'radial-gradient(ellipse at center, rgba(0,0,0,'+(opacity/2)+') 0%, rgba(0,0,0,'+opacity+') 100%)' });
 				}
 			}, 1000);
-			
+
 			// Hide on mouse move
 			$(".overlay_idle").on('click', function(){
 				hideIdleScreen = true;
 				$(this).hide();
 			});
-			
+
 			// Receives and remembers the time when a network request was last made
 			return function(request, sender, response){
 				if (!ConfigManager.alert_idle_start) return true;
```

Comment:
ah but this line should have been the toggle, if user sets it to zero, idle timer will not run